src/main/markdown/makinggwtbetter.md

```diff
@@ -538,6 +538,12 @@ Finally, to actually create an issue is slightly more involved, but most of the
 
 If you have any problems you might want to consult the gerrit documentation on [Uploading Changes](https://gerrit-review.googlesource.com/Documentation/user-upload.html) 
 
+### Gerrit code review<a id="gerritcodereview"></a>
+
+Code review is an essential part of contribution workflow :
+
+Once your code is pushed to gerrit, notifications will be triggered to all reviewers,at this stage your code need jenkins precheck as well +2 in Code-Review to get merged with remote repository,If you are new contributor Jenkins precheck triggering will be followed by +2 in Code-Review by reviewers,whereas existing  contributors will be added to Jenkins Trusted group which allows the jenkins precheck without wait for code review.
```

Comment:
This sentence/sentences needs some formatting :-) . Sometimes missing sometimes to many spaces and a point at the end instead a comma.